source/grid/grid_out.cc

```diff
@@ -3319,12 +3360,12 @@ namespace internal
           std::vector<double> dummy_weights(n_points, 1./n_points);
           Quadrature<1> quadrature1d(boundary_points, dummy_weights);
 
-          // tensor product of points,
-          // only one copy
+          // tensor product of points, only one copy
           QIterated<dim-1> quadrature(quadrature1d, 1);
           q_projector = new Quadrature<dim> (QProjector<dim>::project_to_all_faces(quadrature));
         }
 
+      std::vector<Point<spacedim>> point_cache;
```

Comment:
same here -- could you copy the documentation you added above?

Reply:
fixed